L1Trigger/TrackFindingTracklet/src/TrackletLUT.cc

```diff
@@ -2,18 +2,196 @@
+  //would be useful for consistency.
+
+  unsigned int bendbits = isPS ? 3 : 4;
+
+  std::vector<std::array<double, 2>> bendpars;    // mid, cut
+  std::vector<std::array<double, 2>> bendminmax;  // min, max
+
+  //Initialize array
+  for (int i = 0; i < 1 << bendbits; i++) {
+    bendpars.push_back({{99, 0}});
+    bendminmax.push_back({{99, -99}});
+  }
+
+  //Loop over modules
+  for (auto sm : sensorModules) {
+    int window = sm->windowSize();  //Half-strip units
+    const vector<double>& encodingBend = setup_->encodingBend(window, isPS);
+
+    //Loop over FEbends
+    for (int ibend = 0; ibend <= 2 * window; ibend++) {
+      int FEbend = ibend - window;                                                 //Half-strip units
+      double BEbend = setup_->stubAlgorithm()->degradeBend(isPS, window, FEbend);  //Full strip units
+
+      const auto pos = std::find(encodingBend.begin(), encodingBend.end(), std::abs(BEbend));
+      int bend = std::signbit(BEbend) ? (1 << bendbits) - distance(encodingBend.begin(), pos)
+                                      : distance(encodingBend.begin(), pos);  //Encoded bend
+
+      double bendmin = FEbend / 2.0 - FEbendcut;  //Full Strip units
+      double bendmax = FEbend / 2.0 + FEbendcut;
+
+      //Convert to bendstrip, calculate at module edges (z min, r max) and  (z max, r min)
+      double z_mod[2];
+      double r_mod[2];
+
+      z_mod[0] = std::abs(sm->z()) + (sm->numColumns() / 2 - 0.5) * sm->pitchCol() * sm->cosTilt();  //z max
+      z_mod[1] = std::abs(sm->z()) - (sm->numColumns() / 2 - 0.5) * sm->pitchCol() * sm->cosTilt();  //z min
+
+      r_mod[0] = sm->r() - (sm->numColumns() / 2 - 0.5) * sm->pitchCol() * std::abs(sm->sinTilt());  //r min
+      r_mod[1] = sm->r() + (sm->numColumns() / 2 - 0.5) * sm->pitchCol() * std::abs(sm->sinTilt());  //r max
+
+      for (int i = 0; i < 2; i++) {  // 2 points to cover range in tan(theta) = z/r
+        double CF = std::abs(sm->sinTilt()) * (z_mod[i] / r_mod[i]) + sm->cosTilt();
+
+        double cbendmin =
+            convertFEBend(bendmin, sm->sep(), settings_.sensorSpacing2S(), CF, (layerdisk < N_LAYER), r_mod[i]);
+        double cbendmax =
+            convertFEBend(bendmax, sm->sep(), settings_.sensorSpacing2S(), CF, (layerdisk < N_LAYER), r_mod[i]);
+
+        if (cbendmin < bendminmax[bend][0])
+          bendminmax.at(bend)[0] = cbendmin;
+        if (cbendmax > bendminmax[bend][1])
+          bendminmax.at(bend)[1] = cbendmax;
+      }
+    }
+  }
+  //Convert min, max to mid, cut for ease of use
+  for (int i = 0; i < 1 << bendbits; i++) {
+    double mid = (bendminmax[i][1] + bendminmax[i][0]) / 2;
+    double cut = (bendminmax[i][1] - bendminmax[i][0]) / 2;
+
+    bendpars[i][0] = mid;
+    bendpars[i][1] = cut;
+  }
+
+  return bendpars;
+}
+=======
+TrackletLUT::TrackletLUT(const Settings& settings) : settings_(settings) { nbits_ = 1; }
+>>>>>>> 641a95215dda84fadef167de9c1690d2cd11462e
```

Comment:
@aryd, looks like you pushed without resolving the git conflicts. You'll have to remove everything either between `<<<<<<< HEAD` and `=======` or everything between `=======` and `>>>>>>> 641a95215dda84fadef167de9c1690d2cd11462e`, depending on which is correct.